src/STW.StubApi.Presentation/Services/FieldConfig/Helpers/FieldConfigHelper.cs

```diff
@@ -0,0 +1,35 @@
+namespace STW.StubApi.Presentation.Services.FieldConfig.Helpers;
+
+using Models;
+
+public static class FieldConfigHelper
+{
+    public static FieldConfigDto GetFieldConfigDto(string certType, string commodityCode)
+    {
+        var data = LoadFieldConfigDataFromFile(string.Equals(commodityCode, "03061699") ? "fieldConfigData.json" : "defaultFieldConfigData.json");
+
+        return new FieldConfigDto
+        {
+            CertificateType = certType,
```

Comment:
Should the cert type play a part in deciding what field config to return?

Reply:
This MR is to facilitate testing of a rule that is CHEDP only, I didn't see a need to support multiple cert types for now.